app/.coverage

```diff
@@ -0,0 +1 @@
+!coverage.py: This is a private format, don't read it directly!{"lines":{"/Users/danielmaina/Workspace/python/amity/amity_allocator/app/amity.py":[1,16,19,21,22,24,38,39,40,43,50,60,64,70,73,76,82,85]}}
```

Comment:
Ignore this file. Do not push it to git. It will be generated on the CI when needed.